src/QMCTools/PyscfParser.cpp

```diff
@@ -0,0 +1,202 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Anouar Benali, benali@anl.gov, Argonne National Laboratory
+//
+// File created by: Anouar Benali, benali@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+
+
+
+#include "QMCTools/PyscfParser.h"
+#include <fstream>
+#include <iterator>
+#include <algorithm>
+#include <set>
+#include <map>
+
+
+
+PyscfParser::PyscfParser()
+{
+  basisName = "Gaussian-G2";
```

Comment:
Why leave "Gaussian-G2" here instead of "Gaussian" like everywhere else?

Reply:
Per @anbenali , the basis set name is always available in pyscf, so will be replaced with the actual name.

Reply:
Paul is right, but that tag in that file is mainly a debugging legacy as I was trying to force the code to go through a specific path and did not know where it was picking that data.  I removed the tag from there since it is never read anyways. 

@prckent  and @jtkrogel In the case of multiple basis sets, a single name is impossible. by default I am putting Gaussian (Gaussian-G2 makes no sens). I first wanted to name it "HomeBrew" but I think apple has a trademark on it.. This does not do anything specific except allowing us to know what basis set is used.. any better ideas?